src/main/java/com/google/sps/servlets/UserDetailsServlet.java

```diff
@@ -102,7 +102,7 @@ private User createUserObject(Entity userEntity) throws IOException {
     return recipesList;
   }
 
-private Recipe buildRecipe(Entity recipeEntity) {
+  private Recipe buildRecipe(Entity recipeEntity) {
```

Comment:
Do you still need this function? It seems it could be replaced by a constructor (you could add one more that only takes the entity).